packages/buildcop/buildcop.go

```diff
@@ -131,8 +132,19 @@ See https://github.com/apps/build-cop-bot/.`)
 		}
 	}
 
+	if buildURL == "" {
```

Comment:
May be nice to have test coverage for the option :)

Reply:
Working on a refactor to be able to write tests in https://github.com/googleapis/repo-automation-bots/pull/667.

Reply:
Done (via force push to handle merge conflicts more nicely).